src/app/shared/services/http.service.ts

```diff
@@ -110,47 +117,46 @@ export class HttpService {
       .pipe(catchError(this.handleError()));
   }
 
-  getReports(reportIds: string[], storage: string) {
+  getReports(reportIds: string[], storage: string): Observable<Record<string, CompareReport>> {
     return this.http
-      .get<any>(
-        'api/report/' + storage + '/?xml=true&globalTransformer=' + localStorage.getItem('transformationEnabled'),
-        { params: { storageIds: reportIds } },
-      )
+      .get<
+        Record<string, CompareReport>
+      >('api/report/' + storage + '/?xml=true&globalTransformer=' + localStorage.getItem('transformationEnabled'), { params: { storageIds: reportIds } })
```

Comment:
Could you, instead of using sting concatenation, use a template literal?
For example:
```suggestion
      >(`api/report/${storage}/?xml=true&globalTransformer=${localStorage.getItem('transformationEnabled')}, { params: { storageIds: reportIds } })
```